jlm/rvsdg/region.hpp

```diff
@@ -390,6 +396,17 @@ class region
   [[nodiscard]] static size_t
   NumRegions(const jlm::rvsdg::region & region) noexcept;
 
+  /**
+   * Converts \p region and all of its contained structural nodes with subregions to a tree in
+   * ASCII format.
+   *
+   * @param region The top-level region that is converted
+   * @param annotationMap A map with annotations for region%s or structural_node%s.
```

Comment:
Why are there percentage signs here instead of apostrophes?

Reply:
The doxygen output will look like this:

![Screenshot_2024-07-31_20-41-42](https://github.com/user-attachments/assets/e3d85348-387a-4324-b4c0-645e1ce21240)

The percentage signs ensure that I can create grammatically correct sentences while still getting the links to the data structures.

The "Links to classes" section in the [documentation](https://www.doxygen.nl/manual/autolink.html) gives some more information.